sieve_of_Eratosthenes.py

```diff
@@ -0,0 +1,22 @@
+def sieve_of_eratosthenes(n):
+    
+    prime = [True for i in range(n+1)]
+    p = 2
+    while (p * p <= n):
+        # If prime[p] is not changed, then it is a prime
+        if (prime[p] == True):
```

Comment:
E712 comparison to True should be 'if cond is True:' or 'if cond:'

*Origin: PycodestyleBear (E712), Section: `all.autopep8`.*